app/assets/stylesheets/application.css

```diff
@@ -1,18 +0,0 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
- * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any styles
- * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
- * file per style scope.
- *
- *= require_tree .
- *= require_self
- */
-
-@import "bootstrap-sprockets";
-@import "bootstrap";
```

Comment:
import the variables file and the welcome here. This file should be separated in 4 sections, the vendor files like bootstrap, the variable files, the vendor overrides files and then your own files. Separate this imports using a blank line